fcrepo-audit-triplestore/pom.xml

```diff
@@ -60,7 +60,6 @@
       <dependency>
         <groupId>org.apache.jena</groupId>
         <artifactId>jena-fuseki-embedded</artifactId>
-        <scope>test</scope>
```

Comment:
The test scope is declared in the parent pom, so the test scope here is redundant (which is why it was removed)